cmake/packaging.cmake

```diff
@@ -25,7 +25,7 @@ set(CPACK_PACKAGE_RESOURCE_FILE_README "${CMAKE_CURRENT_SOURCE_DIR}/README.md")
 set(CPACK_DEBIAN_PACKAGE_SHLIBDEPS YES)
 
 # In addition, we depend on gcc for preprocessing
-set(CPACK_DEBIAN_PACKAGE_DEPENDS gcc)
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "gcc, bash-completion")
```

Comment:
Can we get a comment on why we need the extra dependency here?

Reply:
Done.